documentation-site/components/layout.js

```diff
@@ -7,147 +7,58 @@ LICENSE file in the root directory of this source tree.
-    <HeaderNavigation
-      overrides={{
-        Root: {
-          style: ({$theme}) => ({
-            paddingLeft: $theme.sizing.scale800,
-            paddingRight: $theme.sizing.scale800,
-          }),
-        },
-      }}
-    >
-      <NavigationList align={ALIGN.left}>
-        <Block display="flex" alignItems="center">
-          <Link href="/" prefetch>
-            <Block
-              as="img"
-              height="40px"
-              src={Logo}
-              width="101px"
-              overrides={{Block: {style: {cursor: 'pointer'}}}}
-            />
-          </Link>
-          <Block marginLeft="scale600">{version}</Block>
-          <Block
-            overrides={{
-              Block: {
-                style: {
-                  color: 'inherit',
-                  fontStyle: 'italic',
-                },
-              },
-            }}
-            target="_blank"
-            as="a"
-            href="https://github.com/uber-web/baseui/releases"
-          >
-            (Changelog)
-          </Block>
-        </Block>
-      </NavigationList>
-      <NavigationList align={ALIGN.center} />
-      <NavigationList align={ALIGN.right}>
-        <Search />
-        <Block marginLeft="scale600">
-          <ComponentMenu />
-        </Block>
-        <Block
-          $as="a"
-          href="https://github.com/uber-web/baseui"
-          marginLeft="scale600"
-          $style={{textDecoration: 'none'}}
-          target="_blank"
-        >
-          <Button
-            kind={ButtonKind.secondary}
-            overrides={{
-              EndEnhancer: {
-                style: {
-                  marginLeft: 0,
-                },
-              },
-            }}
-            endEnhancer={() => <GithubLogo size={24} color="#276EF1" />}
-          />
-        </Block>
-        <Block
-          $as="a"
-          href="https://join.slack.com/t/baseui/shared_invite/enQtNDI0NTgwMjU0NDUyLTk3YzM1NWY2MjY3NTVjNjk3NzY1MTE5OTI4Y2Q2ZmVkMTUyNDc1MTcwYjZhYjlhOWQ2M2NjOWJkZmQyNjFlYTA"
-          marginLeft="scale600"
-          $style={{textDecoration: 'none'}}
-          target="_blank"
-        >
-          <Button
-            kind={ButtonKind.secondary}
-            overrides={{
-              EndEnhancer: {
-                style: {
-                  marginLeft: 0,
-                },
-              },
-            }}
-            endEnhancer={() => <SlackLogo size={24} color="#276EF1" />}
-          />
-        </Block>
-        <Block marginLeft="scale600">
-          <Link href="/getting-started/installation" prefetch>
-            <Button>Get Started</Button>
-          </Link>
-        </Block>
-      </NavigationList>
-    </HeaderNavigation>
+const SidebarWrapper = styled('div', ({$theme, $isOpen}) => ({
+  display: $isOpen ? 'block' : 'none',
+  paddingTop: $theme.sizing.scale800,
+  marginLeft: $theme.sizing.scale1000,
+  marginRight: $theme.sizing.scale800,
+  '@media screen and (min-width: 820px)': {
```

Comment:
same theming question as above, should this be integrated with the global breakpoints?